05.ls/ls.rb

```diff
@@ -5,11 +5,8 @@
 
 MAX_COLUMN = 3
 
-def main
-  ls_dir = ARGV[0] || '.'
-
-  directory_list = Dir.each_child(ls_dir).to_a
-
+def print_dir_list
+  directory_list = specify_display_content
```

Comment:
specifyには特定するという意味がありますが、「何か特定のひとつ」を見つけて持ってくるイメージがありますね。
実際は複数の値が返ってくるので、ここはfetch（取ってくる）とか、find（見つけてくる）みたいな名前がいいかなーと思います。

あと、`directory_list`、`display_content`、`file_list`というのはおそらく同じものをそれぞれ違う呼び方にしている感があります。
以下の資料に書いたように、同じものには一貫して同じ名前を付けた方がわかりやすいコードになります。

[ユーザーを見つけたら……Aさん「自転車に入れます」Bさん「人間に入れます」C \| FJORD BOOT CAMP（フィヨルドブートキャンプ）](https://bootcamp.fjord.jp/pages/readable-variable-name)

というわけで、名前付けの一例としてはこんな感じでしょうか。

```suggestion
  file_list = fetch_file_list
```

ところで、listには「配列」の意味があり、一般的なコーディングルールとして配列の名前は複数形で表す、というセオリーもあります。それに従うと、`file_list`ではなく`files`と短く表現できます。

```suggestion
  files = fetch_files
```

もしファイルの場合とディレクトリの場合があるからfileともdirectoryとも呼びづらい、という場合はpathを使うといいかもしれません。

```suggestion
  paths = find_paths
```

とか

```suggestion
  paths = list_paths
```

みたいな感じで。

Reply:
コメントありがとうございました｡
いただいたご指摘のように修正いたしました｡